library/src/main/java/org/apache/apex/malhar/lib/state/spillable/SpillableMapImpl.java

```diff
@@ -236,13 +237,27 @@ public void beginWindow(long windowId)
   @Override
   public void endWindow()
   {
+    boolean isTimeUnifiedStore = (store instanceof ManagedTimeUnifiedStateImpl);
```

Comment:
make a class variable can slightly improve the performance. But this flag only apply in endWindow() and it's not really a class variable. I prefer put here